docs/commands.md

```diff
@@ -7,7 +7,7 @@
 Create a new time-series.
 
 ```sql
-TS.CREATE key [RETENTION retentionSecs] [LABELS field value..]
+TS.CREATE key [RETENTION retentionSecs] [CHUNK_SIZE maxSamplesPerChunk] [LABELS field value..]
```

Comment:
We initially thought not to expose chunksize in the docs.  If we specify this, we need a default documentation below (as retenionSecs) and I also wonder if it's not possible to do so in TS.ADD and TS.INCRBY/DECRBY.
Given that we had more PR's on adding this to the docs, I suggest we add it, but than all the way. @gkorland @danni-m

Reply:
Then we should remove `CHUNK_SIZE` from this PR, and after discussing add it in another PR.